coalib/io/FileFactory.py

```diff
@@ -0,0 +1,111 @@
+import linecache
+import os
+
+from coala_utils.decorators import generate_eq
+
+
+@generate_eq('name', 'timestamp')
+class FileFactory:
+    """
+    The FileFactory is a wrapper class to replace the actual file contents
+    from the file-dict. The object contains the following information about
+    the file:
+
+        * The filename (absolute file path).
+        * The modification timestamp of the file.
+        * The file in the form of a string.
+        * The file in the form of a list.
+        * The file in raw format.
+
+    The equality of the object is checked just based on its filename and
+    timestamp.
+
+    To initialize a FileFactory object for a file:
+
+    >>> fp = FileFactory("./tests/io/FileFactoryTestFiles/test.txt")
```

Comment:
Use single quotes, that's the preferred quotation for strings that don't contain single backticks

Reply:
eh single quotes* :3